apricot/local/serviceutil.go

```diff
@@ -84,7 +87,19 @@ func (s *Service) getStringMap(path string) map[string]string {
 			if v.Type() != cfgbackend.IT_Value {
 				continue
 			}
-			theMap[k] = v.Value()
+			if v.Value() != strings.TrimSpace(v.Value()) {
```

Comment:
Better:
`if trimmed = strings.TrimSpace(v.Value()); v.Value != trimmed {`

This way you already have a handle to the trimmed string and you don't need to trim again on line 97.